app/Plugins/Stripe/Controllers/SettingsController.php

```diff
@@ -166,7 +168,7 @@ public function postPaymentWithStripe(Request $request)
                     $payment = new \App\Http\Controllers\Order\InvoiceController();
                     $payment->postRazorpayPayment($invoice);
                     if ($invoice->grand_total && emailSendingStatus()) {
-                        $this->sendPaymentSuccessMailtoAdmin($invoice->currency, $invoice->grand_total, \Auth::user(), $invoice->invoiceItem()->first()->product_name);
+                        $this->sendPaymentSuccessMailtoAdmin($invoice->currency, $invoice->grand_total, $user, $invoice->invoiceItem()->first()->product_name);
```

Comment:
why are we changing this to user?

Reply:
I am using this function for auto-renewal also , from there I cannot get Auth::user() value